File: data/contacts.js
class Contact {
  constructor(type, brand, style, price) {
    this.type = type;
    this.brand = brand;
    this.style = style;
    this.price = price;
  }
}

const contacts = [
  new Contact('daily', 'Acuvue', 'Moist', 56),
  new Contact('daily', 'Alcon', 'AquaComfort Plus', 52),
  new Contact('daily', 'Bausch + Lomb', 'ONEDay', 45),
  new Contact('daily', 'CooperVision', 'Clarity', 60),
  new Contact('weekly', 'Acuvue', 'Oasys', 60),
  new Contact('weekly', 'Bausch + Lomb', 'SofLens', 58),
  new Contact('weekly', 'CooperVision', 'Avaira', 56),
  new Contact('monthly', 'Acuvue', 'Vita', 52),
  new Contact('monthly', 'Alcon', 'Air Optix', 48),
  new Contact('monthly', 'Bausch + Lomb', 'Ultra', 42),
  new Contact('monthly', 'CooperVision', 'Biofininity', 52)
];

File: fake_db/contacts.py
from decimal import Decimal

from pydantic import BaseModel


class Contact(BaseModel):
    schedule: str
    brand: str
    style: str
    price: Decimal


contacts = [
    {
        "schedule": "daily", "brand": "Acuvue", "style": "Moist", "price": 56
    },
    {
        "schedule": "daily", "brand": "Alcon", "style": "AquaComfort Plus", "price": 52
    },
    {
        "schedule": "daily", "brand": "Bausch + Lomb", "style": "ONEDay", "price": 45
    },
    {
        "schedule": "daily", "brand": "CooperVision", "style": "Clarity", "price": 60
    },
    {
        "schedule": "weekly", "brand": "Acuvue", "style": "Oasys", "price": 60
    },
    {
        "schedule": "weekly", "brand": "Bausch + Lomb", "style": "SofLens", "price": 58
    },
    {
        "schedule": "weekly", "brand": "CooperVision", "style": "Avaira", "price": 56
    },
    {
        "schedule": "monthly", "brand": "Acuvue", "style": "Vita", "price": 52
    },
    {
        "schedule": "monthly", "brand": "Alcon", "style": "Air Optix", "price": 48
    },
    {
        "schedule": "monthly", "brand": "Bausch + Lomb", "style": "Ultra", "price": 42
    },
    {
        "schedule": "monthly", "brand": "CooperVision", "style": "Biofininity", "price": 52
    }
]
